### PR TITLE
[QA] Add firs version (happy_path) of API aceptance tests

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers

@flopezag 
#### Description

Acceptance Tests (Happy Path) for each GlanceSync API resource using Apiary mock.
#### Testing

Locally.

### CODE REVIEW

File: tests/acceptance/commons/behave_step_helpers.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    :param image_name (string): Name of the image
+    :return: None
+    """
+
+    glance_ops = context.glance_manager_list[region]
+    images_list = glance_ops.get_images(image_name)
+
+    # There must not be images with the given name
+    assert_that(len(images_list), is_(equal_to(0)),
+                "There are images with the name '{}' in '{}', and it sloudn't".format(image_name, region))
```
Comment (flopezag) [inline tests/acceptance/commons/behave_step_helpers.py]: Add new line at the end of the line.

File: tests/acceptance/features/api/api_sync_region.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  As a GlanceSync API user
+  I want to sync images in a region using the GlanceSync API
+  So that I can have all FIWARE images up to date in all FIWARE nodes.
+
+  GlanceSync API operation under testing:
+    POST regions/regionId
+
+
+  @happy_path @env_dependant @experimentation @skip # needs a real environment.
```
Comment (flopezag) [inline tests/acceptance/features/api/api_sync_region.feature:15]: I would need to undestarnd the different tags. I know skip but what are the others.

Comment (jframos) [inline tests/acceptance/features/api/api_sync_region.feature:15]: In this scenario, I am using a explicit dataset (name of the region) that is particular of our Experimentation environment. This scenario, as it has been described, will not be able to run on other environment if it does not have the same name of regions

```
     @env_dependant @experimentation
```

This scenario is a "Happy Path": a default scenario featuring no exceptional or error conditions, and comprises the sequence of activities executed if everything goes as expected. The tag is very useful to execute only these type of scenarios as part of Smoke Tests or Sanity Checks for the component.

```
      @happy_path
```

Comment (flopezag) [inline tests/acceptance/features/api/api_sync_region.feature:15]: NTC

**Comment by jframos** [comment]
The target location of the config file copied by Jenkins in GlanceSyn-Acceptance job must be modified. The new target is `conf` instead of `config`.

**Pull request merged**